src/Controller/MailTesterController.php

```diff
@@ -74,16 +68,16 @@ public function mailTester(Request $request, SenderInterface $sender): Response
         }
 
         if (isset($mailTester['submit']) && $form->isValid()) {
-            $this->sendMail($mailTester, $sender, $form);
+            $this->sendMail($request, $mailTester, $sender, $form);
         }
 
         return $this->render('@SynoliaSyliusMailTesterPlugin/Admin/MailTester/mail_tester.html.twig', ['form' => $form->createView()]);
     }
 
-    private function sendMail(array $mailTester, SenderInterface $sender, FormInterface $form): void
+    private function sendMail(Request $request, array $mailTester, SenderInterface $sender, FormInterface $form): void
     {
         try {
-            $this->flashBag->add('success', $this->translator->trans('sylius.ui.admin.mail_tester.success'));
+            $request->getSession()->getFlashBag()->add('success', $this->translator->trans('sylius.ui.admin.mail_tester.success'));
```

Comment:
it's a bit strange to add a flashbag that indicate success whereas if an error occured, it'll be after and this flashbag will be displayed too. 

Maybe we should do this around the call of the function, or move this success later ?

Reply:
I hadn't paid attention to that, but indeed, it makes more sense.